tests/user/test_get_user_by_id.py

```diff
@@ -0,0 +1,230 @@
+import pytest
+from allure import feature, description, link, step, title
+from hamcrest import assert_that, is_, contains_string, is_not, empty
+
+from framework.asserts.common import assert_status_code, assert_content_type, assert_response_message
+from framework.asserts.user_asserts import assert_all_user_data_matches
+from framework.endpoints.authenticate_api import AuthenticateAPI
+from framework.endpoints.users_api import UsersAPI
+from framework.tools.generators import generate_user
+from framework.tools.generators import generate_jwt_token
+
+
+@feature("Getting user info by ID")
+@link(
+    url="https://github.com/Sunagatov/Online-Store/wiki/",
+    name="(!) WAIT LINK. Description of the tested functionality",
+)
```

Comment:
I think you can delete `@link`. Difficult to maintain

Reply:
Removed.